requirements_dev.txt

```diff
@@ -0,0 +1,8 @@
+-r requirements_minimal.txt
+
+# Tools used during development, prefer installing these with pre-commit
+black
+pre-commit
+pylint
```

Comment:
pylint cannot be installed with pre-commt

Reply:
"prefer installing these with pre-commit", this is not possible for pylint as it's a system hook.